spec/filters/elasticsearch_spec.rb

```diff
@@ -1 +1,12 @@
+# encoding: utf-8
+
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/plugin"
+
+describe "filter/elasticsearch" do
```

Comment:
do you prefer using the file path instead of the class? is that a "better" thing?